sprint9-objects/ex1.js

```diff
@@ -0,0 +1,23 @@
+//Scrieți o funcție care creează un obiect pentru a păstra informații despre rețeta ta preferată
+//Ar trebui să aibă proprietăți pentru titlu (un șir de caractere), porții (un număr) și ingrediente (o șir de elemente - array).
+//  Va fi afișată pe rânduri separate (o instrucțiune console.log pentru fiecare) toate informațiile după crearea acestuia.
+
+//Exemplul 1:
+//Intrare: titlu = “Pizza”, portii = 3, ingrediente = [“sunca”, “cas”, “ketchup”]
+//Ieșire: { titlu: “Pizza”, portii: 3, ingrediente: [”sunca”, “cas”, “ketchup”] }
+
+function retetaPaste(titlu, portii, ingrediente) {
+    return {
+        titlu: titlu,
+        portii: portii,
+        ingrediente: ingrediente
```

Comment:
```suggestion
        ingrediente: [...ingrediente]
```
nu uitam sa copiam lista, ca sa nu fie doar un pointer